examples/SEA076_20230906T0852_nrt.cdl

```diff
@@ -0,0 +1,389 @@
+		:platform = "sub-surface gliders" ;
+		:processing_level = "L1. Quality control flags from IOOS QC QARTOD https://github.com/ioos/ioos_qc Version: 2.1.0 " ;
+		:project = "SAMBA" ;
+		:project_url = "https://voiceoftheocean.org/samba-smart-autonomous-monitoring-of-the-baltic-sea/" ;
+		:publisher_email = "callum.rollo@voiceoftheocean.org" ;
+		:publisher_name = "Callum Rollo" ;
+		:publisher_url = "https://voiceoftheocean.org" ;
+		:references = "created with pyglider https://github.com/c-proof/pyglider" ;
+		:sea_name = "Baltic" ;
+		:source = "Observational data from a profiling glider." ;
+		:standard_name_vocabulary = "CF STandard Name Table v49" ;
+		:summary = " Part of SAMBA continuous monitoring" ;
+		:time_coverage_end = "2023-09-15T07:37:07.016000000" ;
+		:time_coverage_start = "2023-09-06T08:52:59.375000000" ;
+		:title = "OceanGliders example file for SeaExplorer data" ;
+		:total_dives = 64LL ;
+		:transmission_system = "IRIDIUM" ;
+		string :variables = "time", "latitude", "longitude", "nav_state", "heading", "pitch", "roll", "dive_num", "security_level", "declination", "internal_temperature", "internal_pressure", "desired_heading", "ballast_cmd", "ballast_pos", "linear_cmd", "linear_pos", "angular_cmd", "angular_pos", "voltage", "altimeter", "dead_reckoning", "nav_resource", "conductivity", "temperature", "pressure", "salinity", "chlorophyll", "chlorophyll_raw", "cdom", "cdom_raw", "backscatter_scaled", "backscatter_raw", "oxygen_concentration", "temperature_oxygen", "ad2cp_heading", "ad2cp_pitch", "ad2cp_roll", "ad2cp_pressure", "ad2cp_time", "ad2cp_beam1_cell_number1", "ad2cp_beam2_cell_number1", "ad2cp_beam3_cell_number1", "ad2cp_beam4_cell_number1" ;
+		:wmo_id = "6801665" ;
+		:deployment_start = "2023-09-06T08:52:59.375000000" ;
+		:deployment_end = "2023-09-15T07:37:07.016000000" ;
+		:disclaimer = "Data, products and services from VOTO are provided \'as is\' without any warranty as to fitness for a particular purpose." ;
+		:platform_vocabulary = "https://vocab.nerc.ac.uk/collection/L06/current/27/" ;
+		:contributor_email = "callum.rollo@voiceoftheocean.org, louise.biddle@voiceoftheocean.org" ;
+		:contributor_role_vocabulary = "http://vocab.nerc.ac.uk/search_nvs/W08/" ;
+		:institution_role = "principal investigator " ;
+		:institution_role_vocabulary = "https://vocab.nerc.ac.uk/collection/C86/current/" ;
+		:data_url = "https://erddap.observations.voiceoftheocean.org/erddap/tabledap/nrt_SEA076_M19" ;
+		:rtqc_method = "IOOS QC QARTOD https://github.com/ioos/ioos_qc" ;
+		:rtqc_method_doi = "None" ;
+data:
+
+ CNDC = -0.0041, -0.0037, -0.0036, -0.0035, -0.0037, -0.0036, -0.0037, 
+    -0.0032, -0.0032, -0.0028 ;
+
+ CNDC_QC = 4, 4, 4, 4, 4, 4, 4, 4, 4, 4 ;
+
+ PHASE = 119, 116, 116, 116, 116, 116, 116, 116, 116, 116 ;
```

Comment:
What is the meaning of those values?

Reply:
This is the nav state from the SeaExplorer that indicates e.g. when the glider is in communication with satellite. I've added a comment to this effect. I couldn't find a dive phase variable in the NERC OG vocabulary server. Is there one we should use? All I can find is GLCOMPH which has no indication of what values to use https://vocab.nerc.ac.uk/collection/OG1/current/GLCOMPH/

Reply:
I've removed PHASE until it is defined as part of OG1.0